tests/CMakeLists.txt

```diff
@@ -361,8 +361,15 @@ if (PYTHON_VERSION_MAJOR GREATER 2)
     set_tests_properties("install-vm" PROPERTIES LABELS quick)
 endif()
 
+if (SSG_PRODUCT_RHEL9)
 add_test(
     NAME "automatus-sanity"
     COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/tests/automatus.py" "--help"
 )
 set_tests_properties("automatus-sanity" PROPERTIES LABELS quick)
+
+add_test(
```

Comment:
What is expect behavior when a rule is in more than one component? Is that something we want to support?

Reply:
I think so, for example, consider a service disable rule: If systemd changes or enhances the disable/mask feature, we will want to update those rules because of the systemd invocation. On the other hand, if a component changes its service structure or service names, we will want to update that rule for the other reason.